lib/assets/javascripts/constants.js

```diff
@@ -0,0 +1,15 @@
+(function(ctx){  
+  var vals = {
+    PASSWORD_MIN_LENGTH: 8,
+    PASSWORD_MAX_LENGTH: 128,
+
+    VALIDATION_MESSAGE_PASSWORD: "The password must be between 8 and 128 characters.",
+    VALIDATION_MESSAGE_EMAIL: "You must enter a valid email address.",
+    VALIDATION_MESSAGE_REQUIRED: "This field is required.",
+    VALIDATION_MESSAGE_PASSWORDS_MATCH: "The passwords must match."
+  };
+
+  for(var key in vals){
```

Comment:
Not need to iterate over vals object keys. You could just type ctx = vals; and from outside would be accessed similarly, i.e. dmproadmap.constants.PASSWORD_MIN_LENGTH